cards/src/main/resources/cards/custom/group0/other_death_knight/tgt/minion_invincible.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Invincible",
+  "baseManaCost": 6,
+  "type": "MINION",
+  "heroClass": "SPIRIT",
+  "baseAttack": 1,
+  "baseHp": 1,
+  "rarity": "LEGENDARY",
+  "description": "Immune. Can't gain enchantments.",
```

Comment:
Yikes. What does this really mean? Maybe this one goes to staging, because it interacts with targeting logic in a really funky way.